projects/orquestra-sdk/src/orquestra/sdk/_client/_base/_config/_settings.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+
+from orquestra.workflow_shared.schema.configs import RuntimeConfiguration, RuntimeName
+
+# Why JSON?
+#  The Python TOML package is unmaintained as of 2022-02-18.
+#  It is not compatible with the 1.0 version of the TOML spec:
+#    https://github.com/uiri/toml/issues/267#issuecomment-886139340
+#  YAML is not accepted by Pydantic's parse_file and is unlikely to ever be supported:
+#    https://github.com/samuelcolvin/pydantic/issues/136
```

Comment:
Praise: recording why an implementation decision has been made in the same place that it's implemented is one of my favourite patterns and a great way to prevent future devs wasting time wondering why we did something a particular way!